Find a functionally equivalent code:com.android.keyguard.FaceUnlock.handleReportFailedAttempt:COMMENT
Method Modifier: 
Comment:/**
 * Increments the number of failed Face Unlock attempts.
 */

@@ -4,8 +4,8 @@
     // We are going to the backup method, so we don't want to see Face Unlock again until the
     // next time the user visits keyguard.
     KeyguardUpdateMonitor.getInstance(mContext).setAlternateUnlockEnabled(false);
-    mKeyguardScreenCallback.reportFailedUnlockAttempt();
+    mKeyguardScreenCallback.reportUnlockAttempt(false);
 }

